src/sql/src/parsers/create_parser.rs

```diff
@@ -297,6 +297,15 @@ impl<'a> ParserContext<'a> {
             None
         };
 
+        let eval_interval = if self
+            .parser
+            .consume_tokens(&[Token::make_keyword("EVAL"), Token::make_keyword("INTERVAL")])
+        {
+            Some(self.parse_interval()?)
```

Comment:
Can we restrict the interval so users can only use seconds, hours or days. This avoids handling things like months. We may modify the `parse_interval()` method.

cc @fengys1996

Reply:
FYI:
https://github.com/GreptimeTeam/greptimedb/blob/a678b4dfd64ad0fcb5c3e91c1eed6c4054609318/src/query/src/range_select/plan_rewrite.rs#L163-L175